chapter_computer-vision/kaggle-dog.md

```diff
@@ -439,8 +440,8 @@ train(net, train_iter, valid_iter, num_epochs, lr, wd, devices, lr_period,
 
 ```{.python .input}
 #@tab pytorch
-devices, num_epochs, lr, wd = d2l.try_all_gpus(), 5, 0.001, 1e-4
-lr_period, lr_decay, net = 10, 0.1, get_net(devices)
+devices, num_epochs, lr, wd = d2l.try_all_gpus(), 10, 0.0001, 1e-4
```

Comment:
What did you observe when setting lr=0.005 as in mxnet?

Reply:
With `lr=0.005` in pytorch, the model just didn't converge. Infact the loss kept on increasing for a while then started decreasing. The plot was kind of messy. It was not a strictly decreasing function plot.